kvm/v13.0.0/release.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v13.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.3.0
+  - name: chart-operator
+    version: 0.13.0
```

```suggestion
    version: 0.13.2
```
From @tomahawk28 request here https://github.com/giantswarm/releases/pull/450#issuecomment-698955475 - if it's ok w @MarcelMue !

---

Just FYI we are hitting issues with testing 1.18 - so not sure when this will get merged.

---

thanks @MarcelMue . Do you think we could do a KVM v12.3.0 (https://github.com/giantswarm/giantswarm/issues/13751) w Helm 3 first?

---

I think you can do that - please coordinate with @giantswarm/team-rocket though. I don't see a big issue though